index.html

```diff
@@ -8,9 +8,42 @@
     <link rel="stylesheet"  href="style.css">
 </head>
 <body>
-    <header>
-        
+    <div class ="porfolio_container">
+    
+    <header >
+        <nav >
+            <ul class ="top_navigation">
+                <li class ="nameplate"> 9:41</li>
+                <ui class ="nameplateicons">
+                <li class ="icons"> <img src = "image/Cellular Connection.png"></li>
+                <li class ="icons"> <img src = "image/Wifi.png"></li>
+                <li class ="icons"> <img src = "image/Battery.png"></li>
+            </ui>
+        </nav>
+        <nav >
+            <ul class ="item_navigation">
+                <li class ="item"><span class= "text"><h2>Omotayo Akande</h2></span></li>
+                <li class ="union"><img src = "image/Union.png"></li>
+               
+            </ul>
+    
+    
+        </nav>
     </header>
     
+    <section >
+        <div class ="headline">
+        <h1 class = "primary">Hey there<br>I’m Tayo.</h1> 
+        <h2> I am a Software developer</h2>
+            
+        <h3 class="supportingtext"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</h3>
+
+            <div class = "social_media"> <img src ="image/Social Media.png"></div>
+            <div class = "illustration"><img src = "image/illustration-header-mobile.png"></div>
```

Comment:
- [ ] This image should be added as a background image using CSS not as an image with the (`img`tag) Because it wasn't added as a background image from the screenshot below you can see that it is covering some of the text and not well positioned :100: 
- [ ] Your background color isn't completely filling the entire section. Please do well to fix this issue as well so as to satisfy all the project requirements and make your page look good :+1: 
- [ ] Your design isn't well aligned like that of the Figma design. Please reduce the margin you applied to the sections and try to stick to the Figma design as much as possible :wink: 
- Your design
![image](https://user-images.githubusercontent.com/102699958/212217044-c65fb7b4-c806-4074-b9a9-a2bbbd8cd5d3.png)
- The Figma design
![image](https://user-images.githubusercontent.com/102699958/212217268-67960d4d-1557-405b-9e73-392270207377.png)